src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/MediaStreamControllerTest.php

```diff
@@ -121,6 +121,27 @@ public function testGetImageAction()
         $this->assertSame('image/gif', $client->getResponse()->headers->get('Content-Type'));
     }
 
+    public function testGetImageActionSvg()
+    {
+        $client = $this->createAuthenticatedClient();
+        $filePath = $this->createMediaFile('test.svg', 'sulu.svg');
+        $media = $this->createMedia($filePath, 'Test svg');
+
+        $client->request('GET', $media->getFormats()['small-inset']);
+        $this->assertHttpStatusCode(200, $client->getResponse());
+        $this->assertSame('image/svg+xml', $client->getResponse()->headers->get('Content-Type'));
+
+        $client->request('GET', $media->getFormats()['small-inset.svg']);
+        $this->assertHttpStatusCode(200, $client->getResponse());
+        $this->assertSame('image/svg+xml', $client->getResponse()->headers->get('Content-Type'));
+
+        if (class_exists(\Imagick::class)) {
```

Comment:
Wouldn't it make sense to move that into a separate test method, and skip it if this is not available?

Reply:
Also I'm not sure why to skip this as png should also work on gd but maye travis doesn't support png so maybe you should request `.jpg` instead of `.png` here as jpg seems to work above and as @danrot wrote I would create an own testcase for SvgAsJpg case.